test/funnels/automation/StableDepositorUniV3.t.sol

```diff
@@ -72,9 +78,16 @@ contract StableDepositorUniV3Test is DssTest {
         buffer.approve(USDC, address(depositor), type(uint256).max);
         buffer.approve(DAI,  address(depositor), type(uint256).max);
 
+        refTick = UniV3Utils.getCurrentTick(DAI, USDC, uint24(100));
+        tickLower = refTick - 100;
+        tickUpper = refTick + 100;
```

Comment:
Have you consider just keeping REF_TICK with the name it was, and the lower and upper with the REF_TICK+100 and -100? I think it was clearer and the resulting PR will be much smaller.

Reply:
I reverted the changes to REF_TICK to keep a smaller diff